src/app/partials/nav/header-static/header-static.component.scss

```diff
@@ -3,19 +3,251 @@
 header{
 	position: fixed;
 	left: 0;
-	width: 100%;
 	top: 0;
 	color: white;
 	background-color: rgba(255,255,255,0);
 	width: 100%;
-	padding: 10px 40px;
 	box-sizing: border-box;
 	z-index: 1000;
-	font-size: $fs-14;
 	transition: all 0.2s ease-in-out;
 	.logo{
 		img{
 			width: 30px;
 		}
 	}
 }
+.dark-overlay{
+	width:100%;
+	height:100%;
+	position:fixed;
+	z-index:100;
+	background:rgba(0,0,0,0.7);
+	overflow:hidden;
+}
+.menu-expander{
+	display:none;
+	color: black;
+	cursor:pointer;
+	position:absolute;
+    top:25px;
+    right:10px;
+    z-index:100;
+
+}
+.menu-bar1, .menu-bar2, .menu-bar3 {
+    width: 30px;
+    height: 5px;
+    
+    margin: 6px 0;
+    transition: 0.4s;
+}
+.dark-bg {
+	background: #000;
+}
+.white-bg {
+	background: white;
+}
+
+.menu-change .menu-bar1 {
+    -webkit-transform: rotate(-45deg) translate(-9px, 6px);
+    transform: rotate(-45deg) translate(-9px, 6px);
+}
+
+.menu-change .menu-bar2 {opacity: 0;}
+
+.menu-change .menu-bar3 {
+    -webkit-transform: rotate(45deg) translate(-8px, -8px);
+    transform: rotate(45deg) translate(-8px, -8px);
+}
+
+header *{
```

Comment:
I think we have this css over body, and if not please add this to body. @live-wire

Reply:
Done! Added the css in `base.scss`